benchmarks/Makefile

```diff
@@ -32,14 +32,15 @@ bmarks = \
 	mt-matmul \
 	mt-memcpy \
 	pmp \
+	vec-memcpy \
 
 #--------------------------------------------------------------------
 # Build rules
 #--------------------------------------------------------------------
 
 RISCV_PREFIX ?= riscv64-unknown-elf-
 RISCV_GCC ?= $(RISCV_PREFIX)gcc
-RISCV_GCC_OPTS ?= -DPREALLOCATE=1 -mcmodel=medany -static -std=gnu99 -O2 -ffast-math -fno-common -fno-builtin-printf -fno-tree-loop-distribute-patterns -march=rv$(XLEN)gc -mabi=lp64d
+RISCV_GCC_OPTS ?= -DPREALLOCATE=1 -mcmodel=medany -static -std=gnu99 -O2 -ffast-math -fno-common -fno-builtin-printf -fno-tree-loop-distribute-patterns -march=rv$(XLEN)gcv -mabi=lp64d
```

Comment:
I was going to suggest that we want to turn off autovectorization so that the C code continues to run on implementations without the V extension, but it appears that only happens at `-O3`, so this should be OK.